src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,32 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(actor);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't add actor to DB", e);
```

Comment:
```suggestion
            throw new DataProcessingException("Can't add actor to DB: " + actor, e);
```

This is not an error, just a tip, it's better to add more information to the exception message, it will be easier to look for an error in the logs.

Reply:
thank you for your feedback and advice for improvements!